kafka/0.10/Dockerfile

```diff
@@ -6,13 +6,13 @@ MAINTAINER Debezium Community
 # Set the version, home directory, and MD5 hash.
 # MD5 hash taken from http://kafka.apache.org/downloads.html for this version of Kafka
 #
-ENV KAFKA_VERSION=2.1.1 \
+ENV KAFKA_VERSION=2.2.0 \
     SCALA_VERSION=2.12 \
     CONFLUENT_VERSION=5.1.2 \
```

Comment:
Could you check whether there's an accompanying update of Confluent Platform? This is used for pulling in matching versions of the Avro serializers.

Reply:
@gunnarmorling Jiri and I did check and there is a deserialization issue by upgrading the platform at the moment, see https://issues.jboss.org/browse/DBZ-1237

Reply:
Ok, thanks!